### ISSUE TITLE
Bug: messageView.addToolbarButton does not create a button

### ISSUE THREAD

**Issue by kadimgh** [InboxSDK/InboxSDK#1056]
**Describe the bug**
messageView.addToolbarButton does not create a button, while other button methods (including, but not limited to messageView.addAttachmentsToolbarButton) create button without issues

**To Reproduce**

1. Make sure you're running the latest version of InboxSDK: @inboxsdk/core@2.1.11
2. Write a code that created MessageView Toolbar Button and Attachments Toolbar Button:
```
  sdk.Conversations.registerMessageViewHandler((messageView) => {

    messageView.addToolbarButton({
      title: "MV",
      iconUrl: "https://lh5.googleusercontent.com/itq66nh65lfCick8cJ-OPuqZ8OUDTIxjCc25dkc4WUT1JG8XG3z6-eboCu63_uDXSqMnLRdlvQ=s128-h128-e365",
      onClick(event) {console.log("MV clicked");},
      section: "MORE",
      orderHint: 0
    });

    messageView.addAttachmentsToolbarButton({
        title: "A",
        iconUrl: "https://lh5.googleusercontent.com/itq66nh65lfCick8cJ-OPuqZ8OUDTIxjCc25dkc4WUT1JG8XG3z6-eboCu63_uDXSqMnLRdlvQ=s128-h128-e365",
        onClick(event) {console.log("A clicked");}
    });
    
  });
```
4. Reload extension, open email with attachments

**Expected behavior**
To see both buttons in the message toolbar and attachment toolbar

**Actual behavior**
messageView.addAttachmentsToolbarButton is created as expected, but messageView.addToolbarButton does not seem to work

**Screenshots**
![1](https://github.com/InboxSDK/InboxSDK/assets/7655219/54ee8fd5-b94b-4285-ba3a-bbfcf8873f60)
![2](https://github.com/InboxSDK/InboxSDK/assets/7655219/50f5d4aa-02c5-404e-b20e-e11df15ded23)

**Add any other context about the problem here if anything**

**Comment by adm-bome** [comment]
Inprevous Issue: #797  and PR: #807

selector query (problem):
```js
const selector_2022_11_23 =  'td > div.nH.a98.iY > div.nH.aHU .b7.J-M[aria-haspopup=true]';
```

The solution: (a space before: `.aHU`)

```js
const selector_2022_11_23 = 'td > div.nH.a98.iY > div.nH .aHU .b7.J-M[aria-haspopup=true]';
```

**Comment by adm-bome** [comment]
Mabe replace the content of the `_getOpenMoreMenu` method with this

```js
_getOpenMoreMenu() {
  const selectors = [
    'td > div.nH.a98.iY > div.nH .aHU .b7.J-M[aria-haspopup=true]'
    'td > div.nH.a98.iY > div.nH.aHU .b7.J-M[aria-haspopup=true]',
    'td > div.nH.if > div.nH .aHU .b7.J-M[aria-haspopup=true]'
  ];

  const findOpenMoreMenu = (q) => {
    return q.map(item => {
      return document.body.querySelector(item)
    }).filter(a => a)[0] || null;
  };

  // This will find any message's open more menu! The caller needs to make
  // sure it belongs to this message!
  return findOpenMoreMenu(selectors);
}
```

**Comment by wegry** [comment]
@adm-bome PRs welcome. 

I wonder if `'td > div.nH.if > div.nH .aHU .b7.J-M[aria-haspopup=true]'` is still in use anywhere. When testing your patch, we could see if the selector is present with Google Chat and Meet both disabled. If not, we might remove it.

**Comment by shashikiran-im** [comment]
Hi @adm-bome, 

I tried the solution you suggested by adding space for DOM selector in getOpenMoreMenu() method as below and it is working. 

CODE:
'td > div.nH.a98.iY > div.nH .aHU .b7.J-M[aria-haspopup=true]' 
 
Have you planned to generate PR for the same soon?

**Comment by shashikiran-im** [comment]
Hi **@wegry**,

I had updated the code & generated PR ([link](https://github.com/InboxSDK/InboxSDK/pull/1069) to merge.
I kindly request you to merge the PR ASAP.

**@adm-bome:** Thanks for the solution.

Thanks in Advance,
Shashikiran

**Comment by wegry** [comment]
Closed by #1069.
